.circleci/config.yml

```diff
@@ -103,81 +106,41 @@ jobs:
       - run: echo -e "[ui]\nusername = mbed Test <test@mbed.org>\n" > ~/.hgrc
       - checkout
       - run: pip install -e ./
+
       - run: mbed --version
       - run: py.test test
       - run: mbed toolchain -G GCC_ARM
       - run: mbed target -G K64F
       - run: mbed config -G protocol ssh
+
       - run: mkdir .tests
       - run: cd .tests && mbed new new-test
+      - run: |-
+          cd .tests/new-test/mbed-os
+          git fetch origin refs/pull/6886/head:pr6886
```

Comment:
Fyi: There was one last issue in mbed-os that this testing revealed, and is fixed with https://github.com/ARMmbed/mbed-os/pull/6886.

Once that PR is in, these two lines will need to be updated to origin/master until 5.9.0 is released to make sure the updates are pulled in correctly.